big_tests/tests/websockets_SUITE.erl

```diff
@@ -161,3 +167,10 @@ escape_attrs(Config) ->
         special_chars_helper:check_attr_from_to(Geralt, Carol)
 
     end).
+
+instrumentation_events() ->
+    % because mod_websockets is mongoose_http_handler, it starts http_handler_instrumentation globally.
+    % To avoid conflicts with cases when the module is both mongoose_http_handler and gen_mod,
```

Comment:
`mod_websockets` is not a `gen_mod`, and I think there could be just one callback `instrumentation`, because we really don't want to make the same module a `gen_mod` and e.g. a service, a http handler or a listener.

Reply:
mod_bosh is gen_mod and mongoose_http_handler, so it fails to start complaining we add instrumentation for it twice.